app/services/adal/user-authorization.ts

```diff
@@ -1,10 +1,12 @@
-import { remote } from "electron";
+import * as electron from "electron";
```

Comment:
Why did you change this? What additional objects are you using from "electron" in this class?